core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -685,7 +686,7 @@ portfolio.pending.step2_buyer.recipientsAccountData=Recipients {0}
 portfolio.pending.step2_buyer.amountToTransfer=Amount to transfer
 portfolio.pending.step2_buyer.sellersAddress=Seller''s {0} address
 portfolio.pending.step2_buyer.buyerAccount=Your payment account to be used
-portfolio.pending.step2_buyer.paymentSent=Payment sent
+portfolio.pending.step2_buyer.paymentStarted=Payment started
```

Comment:
Rebase.

Reply:
This line should not be changed from current master. Revert.

Reply:
Again from my workspace below:

![Screenshot from 2023-02-28 16-00-51](https://user-images.githubusercontent.com/20237127/221892675-93c92db0-7e6b-41fe-9fdb-1589a8e7f769.png)

Reply:
Your workspace is clearly outdated or adding changes on top of master which are not part of your core PR.

You should see that all commits in your branch are on top of the latest commit to master, which is https://github.com/haveno-dex/haveno/commit/34b79e779b565d2102a4f32210658aea5c83217a.

Then, probably one of the commits in your branch is making this change on top of master. So that change should be reverted in your local workspace.

Reply:
This commit in your PR makes these modifications it shouldn't: https://github.com/haveno-dex/haveno/pull/545/commits/7bd72febdbee1bc41e4b84be7f267164f2d5a746

I recommend squashing all commits, and then your squashed commit will clearly contain these changes against master.